backend/tracim_backend/lib/utils/translation.py

```diff
@@ -21,12 +21,15 @@ def translator_marker(string: str) -> str:
 class Translator(object):
     """
     Get translation from json file
+    you should provide either valid fallback_lang(true value) or valid
```

Comment:
this comment should be the docstring of the __init__, not the docstring of the class, I think

Reply:
done.